synthea-hiv/uploader/fhir_client.py

```diff
@@ -51,7 +51,10 @@ def __init__(self, base_url: str):
 
   def post_single_resource(self, resource: str, data: Dict[str, str]):
     url = f'{self.base_url}/{resource}'
-    response_ = self.session.post(url, json.dumps(data))
+    s = requests.Session()
```

Comment:
This is to get around the issue here:

https://talk.openmrs.org/t/cannot-persist-session-in-openmrs-nightly-image-has-new-log4j/35438

Reply:
I see, so please add a TODO with a reference to this post such that we know when to change it later.

Reply:
Okay so based on the new conversation on that Talk thread, seems this is fixed in the nightly version. That said, relying on the nightly version of OpenMRS is not great and I prefer that we use the latest release instead. So let's keep it until it is fixed in a release.

Reply:
Fixed by using openmrs:latest image, which contains log4j patch and does not have issues with persisting connections